lua/nvim-treesitter/parsers.lua

```diff
@@ -1743,6 +1743,15 @@ list.ungrammar = {
   maintainers = { "@Philipp-M", "@amaanq" },
 }
 
+list.unison = {
+  install_info = {
+    url = "https://github.com/kylegoetz/tree-sitter-unison",
+    files = { "src/parser.c", "src/scanner.c", "src/maybe.c" },
+    requires_generate_from_grammar = true,
```

Comment:
If you have parser.c requires_generate is not needed

Also, do not include maybe.c, if scanner.c includes it that's fine, but that's a really weird construct anyways